static/styles/zulip.css

```diff
@@ -2018,6 +2035,10 @@ div.focused_table {
     /* For FontAwesome icons used in place of images for some users. */
     font-size: 19px;
     text-align: center;
+
+    &.fa-group {
+        margin-left: 8px;
```

Comment:
I changed this to 9px and added a `top: 3px` here to make the alignment more consistent with that of user mentions.